doc/kernel/format_specifications.rdoc

```diff
@@ -142,77 +156,40 @@ of the formatted field:
 
 ==== Specifiers +b+ and +B+
 
-Convert +argument+ as a binary integer:
-
 The two specifiers +b+ and +B+ behave identically
-except when +0b+ or +0B+ is prefixed.
+except when flag +#+ is used.
 
-Flags (all are allowed):
+Convert +argument+ as a binary integer:
 
-- No flag:
+  sprintf('%b', 1)  # => "1"
+  sprintf('%b', 4)  # => "100"
+  # Prefix '..' for negative value.
+  sprintf('%b', -4) # => "..100"
 
-    sprintf('%b', 1)  # => "1"
-    sprintf('%b', 4)  # => "100"
-    # Prefix '..' for negative value.
-    sprintf('%b', -4) # => "..100"
+Type-specific modifier:
 
-- Flag <tt>#</tt>: Prefix +0b+ or +0B+:
+- '#' flag' (use alternate format):
 
     sprintf('%#b', 1)  # => "0b1"
     sprintf('%#B', 1)  # => "Bb1"
     sprintf('%#b', 4)  # => "0b100"
-    # Embed '..' for negative value.
-    sprintf('%#b', -4) # => "0b..100"
 
-- Space flag: Prefix space for non-negative value,
-  <tt>-</tt> for negative value:
+Other effective modifiers:
```

Comment:
No reason to mention modifiers that are the same for all type specifiers.  We don't need the same redundant links in every type specifier section.

Reply:
Removed.